qdtrack/models/mot/quasi_dense.py

```diff
@@ -8,11 +8,30 @@
 @MODELS.register_module()
 class QuasiDenseFasterRCNN(TwoStageDetector):
 
-    def __init__(self, tracker=None, *args, **kwargs):
+    def __init__(self, tracker=None, freeze_detector=False, is_tao=False,*args, **kwargs):
         self.prepare_cfg(kwargs)
         super().__init__(*args, **kwargs)
         self.tracker_cfg = tracker
 
+        self.freeze_detector = freeze_detector
+        if self.freeze_detector:
+            self._freeze_detector()
+        self.is_tao = is_tao
+
+    def _freeze_detector(self):
+        self.backbone.eval()
+        self.neck.eval()
+        self.rpn_head.eval()
+        self.roi_head.bbox_head.eval()
+        for param in self.backbone.parameters():
```

Comment:
How about writing the following 8 lines into a for loop?

Reply:
like this?